cli/tests/test_rebase_command.rs

```diff
@@ -288,16 +288,17 @@ fn test_rebase_single_revision() {
     insta::assert_snapshot!(stdout, @"");
     insta::assert_snapshot!(stderr, @r###"
     Rebased 1 commits onto destination
-    Rebased 2 descendant commits onto parents of rebased commits
+    Rebased 2 other commits
```

Comment:
Now that I've seen the result, I feel like this might make the user wonder what these other commits are. Maybe even the standard "Rebased 2 descendant commits" would be clearer? Up to you to leave as is or think of something clearer.

Reply:
I've updated it to "Rebased X descendant commits".

Here's the types of rebasing possible after this PR:
- Rebasing target commits onto destination (where destination can actually be a descendant of target commits)
- Rebasing descendant commits of target commits onto parents of target commits
- Rebasing descendant commits of target commits after newly rebased target commits (when destination is descendant of target commits)
- Rebasing descendant commits of destination after rebasing of target commits onto destination

I'm not sure if it's strictly necessary to track all of these, plus printing too much output might also be quite confusing...